### ISSUE TITLE
Not able to match sound mode, returning raw sound mode.

### ISSUE THREAD

**Issue by Mace404** [ol-iver/denonavr#57]
Hi,

I got a Denon AVR-X2200W and I got the "Not able to match sound mode, returning raw sound mode." warning in the HA log.

Here are the states:
```
sound_mode: DTS Neural:X
sound_mode_list: MUSIC,MOVIE,GAME,AUTO,VIRTUAL,PURE DIRECT,DOLBY DIGITAL,MCH STEREO,STEREO
sound_mode_raw: DTS Neural:X
```

**Comment by starkillerOG** [comment]
@Mace404 I included the "DTS Neural:X" to the sound mode list in [this PR](https://github.com/scarface-4711/denonavr/pull/52).
As soon as that is merged, this issue will be fixed.

**Comment by Mace404** [comment]
@starkillerOG great, thanks.

**Comment by ol-iver** [comment]
Corrected in version 0.7.5
